pkg/metrics/metrics.jsx

```diff
@@ -842,25 +843,62 @@ class CurrentMetrics extends React.Component {
     }
 }
 
-const SvgGraph = ({ data, resource, have_sat }) => {
+const SvgGraph = ({ minute, data, resource, have_sat }) => {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox={ "0 0 2 " + SVG_YMAX } preserveAspectRatio="none">
+            <Polygons data={data} type="use" resource={resource} have_sat={have_sat} />
+            {have_sat && <Polygons data={data} type="sat" resource={resource} have_sat />}
+        </svg>
+    );
+};
+
+const Polygons = ({ data, type, resource, have_sat }) => {
+    const transform = type == "sat" ? "1,0,0,-1,1," : have_sat ? "-1,0,0,-1,1," : "-2,0,0,-1,2,";
+    const key = type + "_" + resource;
+    const dataFiltered = data.filter(d => d && typeof d[key] === 'number');
+
+    if (!dataFiltered.length)
+        return null;
+
+    const isOverThreshold = dataFiltered.some(sample => sample[key] >= LOAD_THRESHOLD);
     const dataPoints = key => (
         "0,0 " + // start polygon at (0, 0)
-        data.map((samples, index) => (samples && typeof samples[key] === 'number') ? samples[key].toString() + "," + index.toString() : "").join(" ") +
-        " 0," + (data.length - 1) // close polygon
+        dataFiltered.map((samples, index) => Math.min(LOAD_THRESHOLD, samples[key]).toString() + "," + index.toString()).join(" ") +
```

Comment:
In both you and Garrett's screenshots you notice that the normal part of the graph has a slope (lines between actual data points), but now they have a crease at the 0.8 critical point. This makes them look rather weird.

I.e. it's necessary to compute a proper trapezoid/polygon for both sides which keep the slope, instead of just "inventing" a data point at LOAD_THRESHOLD.